source/wrapper/contracts/Wrapper.sol

```diff
@@ -148,7 +148,9 @@ contract Wrapper is Ownable {
 
       // Transfer ether to the user.
       // solium-disable-next-line security/no-call-value
```

Comment:
can remove the  // solium-disable-next-line security/no-call-value now

Reply:
It's still doing a call value, and so is still considered a security flaw as you pass the recipient contract access to all gas and allow reentrancy. Solium will still warn of this even if we catch the bool as you can do reentrancy as then succeed.

Reply:
Interesting, ok.